packages/react/src/components/Breadcrumb/Breadcrumb.Skeleton.js

```diff
@@ -10,19 +10,20 @@ import { settings } from 'carbon-components';
 
 const { prefix } = settings;
 
-export default class BreadcrumbSkeleton extends React.Component {
-  render() {
-    const item = (
-      <div className={`${prefix}--breadcrumb-item`}>
-        <span className={`${prefix}--link`}>&nbsp;</span>
-      </div>
-    );
-    return (
-      <div className={`${prefix}--breadcrumb ${prefix}--skeleton`}>
-        {item}
-        {item}
-        {item}
-      </div>
-    );
-  }
-}
+const BreadcrumbSkeleton = () => {
+  const item = (
```

Comment:
Since this is static content, should we move it outside of the function block scope? Trying to think through when it might be best to versus not 🤔